app/uk/gov/hmrc/formpproxy/sdlt/models/agents/DeletePredefinedAgentModels.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.formpproxy.sdlt.models.agents
+
+import play.api.libs.json.{Json, OFormat}
+
+case class DeletePredefinedAgentRequest(
+  storn: String,
+  agentReferenceNumber: String
+)
+
+object DeletePredefinedAgentRequest {
+  implicit val format: OFormat[DeletePredefinedAgentRequest] = Json.format[DeletePredefinedAgentRequest]
+}
+
+case class DeletePredefinedAgentReturn(
```

Comment:
change **DeletePredefinedAgentReturn** to **DeletePredefinedAgentResponse** as it doesn't have anything to do with returns